instrumentation/runtime-metrics/library/jetty.md

```diff
@@ -0,0 +1,16 @@
+# Jetty Metrics
+
+Here is the list of metrics based on MBeans exposed by Jetty.
+
+| Metric Name                    | Type          | Attributes   | Description                                          |
+|--------------------------------|---------------|--------------|------------------------------------------------------|
+| jetty.session.sessionsCreated  | Counter       | resource     | The number of sessions established in total          |
+| jetty.session.sessionTimeTotal | UpDownCounter | resource     | The total time sessions have been active             |
+| jetty.session.sessionTimeMax   | UpDownCounter | resource     | The maximum amount of time a session has been active |
```

Comment:
since doesn't seem aggregatable
```suggestion
| jetty.session.sessionTimeMax   | Gauge | resource     | The maximum amount of time a session has been active |
```